### ISSUE TITLE
Repetition is difficult to read.

### ISSUE THREAD

**Issue by theFeu** [NETWAYS/gitlab-training#243]
They're pretty squished together and not super readable.
<img width="1206" alt="image" src="https://github.com/NETWAYS/gitlab-training/assets/22217062/4f8a93b1-e266-4568-88a7-ef87753e2309">

One solution I could think of would be splitting them into two slides?

**Comment by theFeu** [comment]
I don't think we can be any more concice without confusing people on what to do.
Kind of leaning towards splitting them up again, which would also reduce the length of time where the trainees have to work by themselves in one go.

**Comment by martialblog** [comment]
Fine for now
